stdlib/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
I think it would be worth adding back the static lib option here to make it easier for folks to consume this library as a native dependency (in conjunction with the .def and .h files under the include folder).
```
[lib]
crate-type = ["staticlib","rlib"]
```